core/base-service/base.spec.js

```diff
@@ -343,59 +331,7 @@ describe('BaseService', function() {
           queryParams: [],
         },
       })
-
-      const [first, second, third] = examples
-      expect(first).to.deep.equal({
-        title: 'DummyService',
-        example: {
-          pattern: '/foo/:world',
-          namedParams: { world: 'World' },
-          queryParams: {},
-        },
-        preview: {
-          label: 'cat',
-          message: 'Hello namedParamA: foo with queryParamA: bar',
-          color: 'lightgrey',
-          namedLogo: undefined,
-          style: undefined,
-        },
-        keywords: ['hello'],
-        documentation: undefined,
-      })
-      expect(second).to.deep.equal({
-        title: 'DummyService',
-        example: {
-          pattern: '/foo/:namedParamA',
-          namedParams: { namedParamA: 'World' },
-          queryParams: {},
-        },
-        preview: {
-          label: 'cat',
-          message: 'Hello namedParamA: foo with queryParamA: bar',
-          color: 'lightgrey',
-          namedLogo: undefined,
-          style: undefined,
-        },
-        keywords: ['hello'],
-        documentation: undefined,
-      })
-      expect(third).to.deep.equal({
-        title: 'DummyService',
-        example: {
-          pattern: '/foo/:world',
-          namedParams: { world: 'World' },
-          queryParams: { queryParamA: '!!!' },
-        },
-        preview: {
-          color: 'lightgrey',
-          label: 'cat',
-          message: 'Hello namedParamA: foo with queryParamA: bar',
-          namedLogo: undefined,
-          style: undefined,
-        },
-        keywords: ['hello'],
-        documentation: undefined,
-      })
+      expect(examples).to.have.lengthOf(1)
```

Comment:
I was thinking it might be helpful to have a comment here indicating that the meat of the examples testing can be found over in `transform-example.spec.js`. Thoughts?

Reply:
Yes, that's a good idea.

Reply:
Alrighty I've added the comment and approved. Will hold off on merging this though in case you want to tweak the comment text

Reply:
Think that works for now!